config/locales/en.yml

```diff
@@ -54,16 +54,28 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    admin_deleted_work_notification:
+      subject: "[%{app_name}] Your work has been deleted by an Admin"
+      hello: "Dear %{user},"
+      opendoors: "contact Open Doors"
+      abuse_committee: "contact our Abuse Committee"
+      bye: "Attached is a copy of your work for your reference."
+      html:
+        part1: "Your work <i>%{title}</i> was deleted from the Archive by a site admin."
+        part2: "If your work was part of an import project managed by our Open Doors team, please %{opendoors_link} with any further questions."
+        part3: "If you have questions, please %{abuse}."
```

Comment:
Shouldn't this be `If it's possible your work violated the Archive's Terms of Service, please %{abuse}.`? Also, nitpicky, but since we're calling the part that links to Open Doors' contact form `opendoors_link`, maybe this should be `abuse_link`?